src/test/java/com/jcabi/aether/ClasspathTest.java

```diff
@@ -94,6 +94,35 @@ public void buildsClasspath() throws Exception {
     }
 
     /**
+     * Classpath can build a classpath with more scopes.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void buildsClasspathWithMoreScopes() throws Exception {
+        MatcherAssert.assertThat(
+            new Classpath(
+                this.project(
+                    this.dependency(
+                        ClasspathTest.GROUP, ClasspathTest.GROUP, "4.12"
+                    )
+                ), this.temp.newFolder(), JavaScopes.TEST, JavaScopes.COMPILE
+            ),
+            Matchers.<File>hasItems(
+                Matchers.hasToString(
+                    Matchers.endsWith(
+                        String.format(
+                            // @checkstyle MultipleStringLiterals (2 lines)
+                            "%sas%<sdirectory",
+                            System.getProperty("file.separator")
+                        )
+                    )
+                ),
+                Matchers.hasToString(Matchers.endsWith("junit-4.12.jar"))
```

Comment:
@amihaiemil please read http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-data-coupling